<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AudioPlayer</class>
 <widget class="QMainWindow" name="AudioPlayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>483</width>
    <height>295</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Times New Roman</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>AudioPlayer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/new/prefix1/resources/images/title.jpg</normaloff>:/new/prefix1/resources/images/title.jpg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_close">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>250</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string> Close</string>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/new/prefix1/resources/images/close.png</normaloff>:/new/prefix1/resources/images/close.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_open_file">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>250</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Open File</string>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/new/prefix1/resources/images/file.png</normaloff>:/new/prefix1/resources/images/file.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>81</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>音频播放</string>
    </property>
   </widget>
   <widget class="QLabel" name="audioName">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>250</y>
      <width>181</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>461</width>
      <height>221</height>
     </rect>
    </property>
    <widget class="QSlider" name="progress_time">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>421</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="progress_volumn">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>170</y>
       <width>341</width>
       <height>21</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_muted">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_seek_backward">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>90</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_stop">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>90</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_player">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>90</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_pause">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>90</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_seek_feedward">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>90</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="label_value_1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>00:00:00</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_value_2">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>50</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>00:00:00</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
